### PR TITLE
Add ORES scores

### PR DESCRIPTION
**Pull request by Ladsgroup**
I have tested functions I added but I couldn't test it in local app. 

Bug: T139009

### CODE REVIEW

**Comment by kaldari** [comment]
I'm not sure it really makes sense to have a whole column devoted to the ORES score, especially since ORES isn't tuned to detect plagiarism yet. I think it would make more sense to list it in the Diff column, under the timestamp, and perhaps only if the score is significant.

**Comment by Niharika29** [comment]
@Ladsgroup thank you for this patch!
I will merge it after the Phabricator ticket gets discussed in our team meeting. 
We will also need some way to let people know what the ORES score actually is but that can come in a separate patch and will need some design input.

**Comment by Ladsgroup** [comment]
Hey Niharika! Thank you. I implemented Ryan's idea and put ORES scores under timestamp. It would be great!

**Comment by MusikAnimal** [comment]
@Ladsgroup sorry, I merged another branch and I guess it conflicts with this one. Should be easy to resolve, let me know if you need help!

**Comment by Ladsgroup** [comment]
I fixed the conflicts :)

**Comment by kaldari** [comment]
@Ladsgroup: We discussed this patch at our sprint planning meeting today and we decided that we only want to show the ORES score if it's a very high score, i.e. likely to be vandalism. The reason for this is that we are also going to be showing the Turnitin scores (https://phabricator.wikimedia.org/T139209), and we don't want to confuse people with a lot of different scores unless it's likely to be important information.

File: public_html/templates/index.html [hunk 1]
```diff
@@ -35,6 +35,12 @@
 						<small class="diff-div-timestamp">
 							<time datetime="{{ row.diff_timestamp }}">{{ row.diff_timestamp }}</time>
 						</small>
+						{% if row.oresScore is not empty %}
+						<br/>
+						<small class="diff-div-ores">
+							ORES score: {{ row.oresScore }}
```
Comment (kaldari) [inline public_html/templates/index.html]: ... and move the '%' here.

File: public_html/templates/index.html [hunk 2]
```diff
@@ -35,6 +35,12 @@
 						<small class="diff-div-timestamp">
 							<time datetime="{{ row.diff_timestamp }}">{{ row.diff_timestamp }}</time>
 						</small>
+						{% if row.oresScore is not empty %}
```
Comment (kaldari) [inline public_html/templates/index.html]: Let's put a threshold here so that we're only showing it for high scores...

**Comment by Ladsgroup** [comment]
Done

**Pull request merged**